demo-angular/package.json

```diff
@@ -0,0 +1,54 @@
+{
+    "nativescript": {
+        "id": "org.nativescript.demo",
```

Comment:
@jannotabamo same here.

Reply:
@msaelices i forgot to mention. I'm done changing it also.